source/solvers/heat_transfer.cc

```diff
@@ -754,6 +760,26 @@ HeatTransfer<dim>::attach_solution_to_output(DataOut<dim> &data_out)
     }
 }
 
+template <int dim>
+double
+HeatTransfer<dim>::calculate_delta_T_ref()
+{
+  double previous_solution_maximum, previous_solution_minimum;
+  if (is_steady(simulation_parameters.simulation_control.method))
+    {
+      previous_solution_maximum = this->present_solution.max();
+      previous_solution_minimum = this->present_solution.min();
+    }
+  else
+    {
+      previous_solution_maximum = this->previous_solutions[0].max();
+      previous_solution_minimum = this->previous_solutions[0].min();
```

Comment:
In some case it's present and in some case it'S previous. Call it solution_maximum and solution_minimum instead of previous then.